web/html/src/manager/admin/setup/products/products.tsx

```diff
@@ -248,6 +248,37 @@ class ProductsPageWrapper extends React.Component {
       .catch(currentObject.handleResponseError);
   };
 
+  addOptionalChannels = (product, channels) => {
+    const currentObject = this;
```

Comment:
I'm not sure if this is necessary? Unless I'm missing something, using `this` should work fine everywhere in the below code block, e.g. we already have `this.refreshServerData()` etc.